packages/graphql/src/services/resolver/resolver.service.ts

```diff
@@ -0,0 +1,98 @@
+import { IResolvers } from '@graphql-tools/utils/Interfaces';
+import { Injectable } from '@pequehq/di';
+
+import {
+  ResolverFieldsMetadata,
+  ResolverMutationsMetadata,
+  ResolverParametersMetadata,
+  ResolverQueriesMetadata,
+} from '../../constants/metadata.constants';
+import { IResolverParamType, IResolverServiceMetadata, ResolverDeclaration } from '../../interfaces';
+import { ResolverStorage } from '../resolver-storage/resolver-storage.service';
+
+@Injectable()
```

Comment:
I wonder if would make sense to completely drop the DI library dependency, and let consumers decide whether to put this service into a container or not 🤔 
If someone wants to go with another IoC container, they would still have to install our `@pequehq/di` to make this work or they would get an error. Do you see any issue removing it?

Reply:
I'd say I don't see issues, initially, I thought this way for the integration with the Framework, but we can "manually" `DI.set` this service, so yeah, we can do it.